0.3.0 (#23)
nestdotland/docs: docs/eggs/publishing-a-module.md

For more details, please refer to [configuration](configuration.md#field-information).

Additional options:
```
-Y, --yes         Disable confirmation prompts.
-d, --dry-run     No changes will actually be made, reports the details of what would have been published.
```

## Programmatically

You just need to import the `publish` function from the latest version of eggs.

```ts
import { publish } from "https://x.nest.land/eggs@0.3.0/src/commands/publish.ts"

const config = {
    "description": "Your brief module description",
    "version": "0.0.1",
    /* ... */
};

publish(config, "my-module");
```

> It is the same function as the CLI, features such as logging are still available.

Reviewer: ```suggestion
This functions identically to the CLI, and features such as logging are still available.
```

> This avoids the need to install eggs. Moreover, every contributor to the project has the same version of eggs.

Reviewer: Not sure what the "Moreover, every contributor to the project has the same version of eggs." means 😅

Author: You can probably put it better than me haha
What I mean is that since with Deno, there is no package.json, the authors can't make sure that all the developers of a module are using the same version of eggs. Therefore, publishing a module in typescript ensures that the version is fixed.

Reviewer: Ahhh, this makes sense now. Thanks for the explanation 🙂
